src/index.js

```diff
@@ -240,7 +240,7 @@ Debug.extend(exported, {isSafari, getPerformanceMetrics: PerformanceUtils.getPer
  * @param {Object} [options]
  * @param {boolean} [options.failIfMajorPerformanceCaveat=false] If `true`,
  *   the function will return `false` if the performance of Mapbox GL JS would
- *   be dramatically worse than expected (e.g. a software WebGL renderer would be used).
+ *   be dramatically worse than expected (a software WebGL renderer would be used).
```

Comment:
nit: remove extra space
```suggestion
 *  the function will return `false` if the performance of Mapbox GL JS would
 *  be dramatically worse than expected (a software WebGL renderer would be used).
```